starter-code/views/drones/index.ejs

```diff
@@ -0,0 +1,9 @@
+----
+<ul>
+  <% drones.forEach((aDrone) => { %>
+    <li><%= aDrone.droneName %></li>
+  <% }) %>
+</ul>
+<form  method='GET' action="new">
+  <button> Add new one </button>
```

Comment:
👎 the button redirection doesn't work! it gives out a 404 error. please take a look at this